pkg/kubelet/container/event.go

```diff
@@ -40,7 +40,16 @@ const (
 	NodeNotSchedulable      = "NodeNotSchedulable"
 	StartingKubelet         = "Starting"
 	KubeletSetupFailed      = "KubeletSetupFailed"
+	FailedDetachVolume      = "FailedDetachVolume"
```

Comment:
Why differentiate between 'device' and 'volume' here?  I understand the specificity, but I'm not sure it will be exceptionally easy for users to understand -- it's possible to provide too much detail in user-facing events.